e2-studio-star-rx72n-firmware/build-wsl.sh

```
@@ -0,0 +1,120 @@
+done
+
+# Clean if requested
+if [ "$CLEAN_FIRST" = "yes" ]; then
+    echo -e "${YELLOW}Cleaning build artifacts...${NC}"
+    make clean
+    echo -e "${GREEN}✓ Clean complete${NC}"
+fi
+
+# Build
+echo -e "${YELLOW}Building firmware (target: $MAKE_TARGET, flags: $MAKE_FLAGS)...${NC}"
+if make $MAKE_FLAGS $MAKE_TARGET; then
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)